files/logstash-configs/22-redmine.conf

```diff
@@ -23,6 +23,9 @@ filter {
         match => {"message" => "%{REDMINE_SUCCESSFUL_LOGIN}"}
         add_tag => "successful_login"
       }
+      date {
+        match => ["redmine_auth_timestamp", "yyyy-MM-dd HH:mm:ss"]
```

Comment:
Let's collapse it then. At this point I can't imagine why we'd want the fields separate, especially because we're already tagging auth versus failure, so we can easily refine search results.